ReactAndroid/src/main/java/com/facebook/react/views/swiperefresh/SwipeRefreshLayoutManager.java

```diff
@@ -82,6 +83,21 @@ public void run() {
     });
   }
 
+  @ReactProp(name = "progressViewOffset", defaultFloat = 0)
+  public void setProgressViewOffset(final ReactSwipeRefreshLayout view, final float offset) {
+    // Use `post` to get progress circle diameter properly
+    // Otherwise returns 0
+    view.post(new Runnable() {
+      @Override
+      public void run() {
+        int diameter = view.getProgressCircleDiameter();
+        int start = Math.round(PixelUtil.toPixelFromDIP(offset)) - diameter;
+        int end = Math.round(PixelUtil.toPixelFromDIP(offset+48));
```

Comment:
Added REFRESH_TRIGGER_DISTANCE

I don't know if there is a better way, those are my firts lines on android so may be i'm not the best to respond that :) BUT i don't find nothing in documentation:

http://developer.android.com/intl/es/reference/android/support/v4/widget/SwipeRefreshLayout.html

Why 48? I don't remember where i saw that but seems the closets to the original.